pages/registration/financial-aid.vue

```diff
@@ -1,11 +1,191 @@
 <template>
-    <p>Registration financial aid</p>
+    <i18n-page-wrapper>
+        <core-h1 :title="$t('title')"></core-h1>
+        <two-col-wrapper>
+            <template #default>
+                <p>{{ $t('coreValueTitle') }}</p>
+            </template>
+            <template #right-col>
+                <p class="content paragraph">
+                    <ext-link
+                        href="https://pyfound.blogspot.com/2017/10/psfs-october-board-meeting.html"
+                        highlight
+                        underline
+                    >
+                        {{ $t('everybodyContributes') }}
+                    </ext-link>
+                    {{ $t('coreValueContent')[0] }}
+                </p>
+                <p class="content paragraph">
+                    {{ $t('coreValueContent')[1] }}
+                </p>
+            </template>
+        </two-col-wrapper>
+        <core-hr></core-hr>
+        <two-col-wrapper
+            v-for="(field, i) in $t('plainTextFields')"
+            :key="`field_${i}`"
+        >
+            <template #default>
+                <p>{{ field.title }}</p>
+            </template>
+            <template #right-col>
+                <p class="content">
+                    {{ field.content }}
+                </p>
+            </template>
+        </two-col-wrapper>
+        <two-col-wrapper>
+            <template #default>
+                <p>{{ $t('visaTitle') }}</p>
+            </template>
+            <template #right-col>
+                <p class="content">
+                    {{ $t('visaContent') }}
+                    <ext-link
+                        href="mailto:organizers@pycon.tw"
+                        highlight
+                        underline
+                    >
+                        {{ $t('orgEmail') }}
+                    </ext-link>
+                </p>
+            </template>
+        </two-col-wrapper>
+        <core-hr></core-hr>
+        <two-col-wrapper>
+            <template #default>
+                <p>{{ $t('applicationHowTo') }}</p>
+            </template>
+            <template #right-col>
+                <div
+                    class="application-how-to-diagram content"
+                    :style="getImage"
+                ></div>
+            </template>
+        </two-col-wrapper>
+        <two-col-wrapper>
+            <template #default>
+                <p>{{ $t('remarkTitle') }}</p>
+            </template>
+            <template #right-col>
+                <ul class="list-disc">
+                    <li class="content">
+                        {{ $t('fill') }}
+                        <ext-link
```

Comment:
The link can be inserted with [slot syntax](https://kazupon.github.io/vue-i18n/guide/interpolation.html#slots-syntax-usage)